test/unit/dao/FeiDao.test.ts

```diff
@@ -97,6 +98,52 @@ describe('FeiDAO', function () {
     it('timelock', async function () {
       expect(await feiDAO.timelock()).to.be.equal(timelock.address);
     });
+
+    it('rollback deadline', async function () {
+      expect((await feiDAO.ROLLBACK_DEADLINE()).toString()).to.be.equal(
+        '1635724800'
+      );
+    });
+
+    it('old governor', async function () {
```

Comment:
nit: the descriptions should tell you what the test is doing. this computes to: "FeiDAO old governor" (old governor...what?)

better: "FeiDAO old governor address is correct"